Dokan/Dokan/Scenes/CategoryList/categoryListCell/CategoryListCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CategoryListCell.swift
+//  Dokan
+//
+//  Created by heba isaa on 17/08/2022.
+//
+
+import UIDokan
+import UIKit
+
+class CategoryListCell: UICollectionViewCell, IdentifiableView {
```

Comment:
I prefer renaming to `CategoryCollectionViewCell`

Reply:
No need for `IdentifiableView` since we've already added a default implementation